src/shared/dto/file-upload.dto.ts

```diff
@@ -0,0 +1,4 @@
+export interface FileUploadData {
+  fileName: string;
+  file: string; // Base64 encoded file content
+}
```

Comment:
Does it make sense to base64 encode the file buffer and directly decode it again afterwards?
(see `KycService. updateFileData`)